spec/PeriodicBackgroundSync-index.bs

```diff
@@ -172,19 +179,21 @@ The <code><dfn method for=PeriodicSyncManager title="register(tag, options)">reg
 1. if the {{PermissionState}} for a {{PermissionDescriptor}} with {{name}} <code>"periodic-background-sync"</code> is not {{granted}}, [=reject=] |promise| with a {{NotAllowedError}} and abort these steps.
 1. Let <var>currentRegistration</var> be the [=periodicsync registration=] in |serviceWorkerRegistration|'s [=list of periodicSync registrations=] whose [=tag=] equals <var>tag</var> if it exists, else null.
 1. If |currentRegistration| is null:
-    1. Let |newRegistration| be a new [=periodicSync registration=].
+    1. Let |newRegistration| be a new [=periodicsync registration=].
```

Comment:
Inconsistency: periodicSync vs. periodicsync. Please make sure we use the same everywhere.

Reply:
for "periodicsync registration", I'm not following camel case, since it's not referring to anything from the interface. I'm consistently following this style for registrations.
for periodicSync events, I'm using camelcase because the event is  part of the idl and can be referred to that. Does that make sense?

Reply:
But you call it `list of periodicSync registrations` a few lines prior to this and still refer to it w/ a capital on line :133 too...

Reply:
I have fixed this to: "list of periodicsync registrations"